### PR TITLE
Reorg and changed namespaces

### PR DESCRIPTION
**Pull request by Tahmid11**

### CODE REVIEW

**Comment by AnthonyEdmonds** [comment]
Why have you done this? What is the benefit?

File: tests/Unit/ActivityHelper/FormatRecipientsTest.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace NetworkRailBusinessSystems\ActivityLog\Tests\Unit\ActivityHelper;
 
-use NetworkRailBusinessSystems\ActivityLog\ActivityHelper;
+use NetworkRailBusinessSystems\ActivityLog\app\Helpers\ActivityHelper;
```
Comment (AnthonyEdmonds) [inline tests/Unit/ActivityHelper/FormatRecipientsTest.php:5]: Incorrect namespace

File: tests/Unit/ActivityHelper/LogNotificationTest.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
 namespace NetworkRailBusinessSystems\ActivityLog\Tests\Unit\ActivityHelper;
 
 use NetworkRailBusinessSystems\ActivityLog\Activity;
-use NetworkRailBusinessSystems\ActivityLog\ActivityHelper;
+use NetworkRailBusinessSystems\ActivityLog\app\Helpers\ActivityHelper;
```
Comment (AnthonyEdmonds) [inline tests/Unit/ActivityHelper/LogNotificationTest.php:6]: Incorrect namespace

File: tests/Unit/ActivityHelper/LogRoleChangeTest.php [hunk 3]
```diff
@@ -3,7 +3,7 @@
 namespace NetworkRailBusinessSystems\ActivityLog\Tests\Unit\ActivityHelper;
 
 use NetworkRailBusinessSystems\ActivityLog\Activity;
-use NetworkRailBusinessSystems\ActivityLog\ActivityHelper;
+use NetworkRailBusinessSystems\ActivityLog\app\Helpers\ActivityHelper;
```
Comment (AnthonyEdmonds) [inline tests/Unit/ActivityHelper/LogRoleChangeTest.php:6]: Incorrect namespace

File: resources/views/activity.blade.php
Comment (AnthonyEdmonds) [inline resources/views/activity.blade.php:1]: File in wrong location

File: config/activity-log.php
Comment (AnthonyEdmonds) [inline config/activity-log.php:1]: File in wrong location

**Review by AnthonyEdmonds — CHANGES_REQUESTED** [review]
I do not know why you have done any of this. The only structure change you were asked to make was to put the tests into folders named after the class they relate to, unless they are a single test for the whole class.

**Review by calum-gray — CHANGES_REQUESTED** [review]
yeah idk why you did any of this

**Pull request closed without merging**